src/java.base/share/man/java.1

```diff
@@ -2975,6 +2975,7 @@ different JDK version.
 .RE
 .TP
 .B \f[CB]\-XX:+ExtendedDTraceProbes\f[R]
+Deprecated. Use combination of these flags instead: -XX:+DTraceMethodProbes, -XX:+DTraceAllocProbes, -XX:+DTraceMonitorProbes
```

Comment:
When a flag is deprecated we move it to the "Deprecated Java Options" section of the manpage.

Reply:
Thank you @dholmes-ora